### DISCUSSION TITLE
Is project still alive ?

### DISCUSSION THREAD

**Discussion by bennukem** [WatchItDev/watchit-app#174] [General]
last update seam pretty old.
Time to pass the project to someone else ?

**Comment by geolffreym** [comment]
Hey @bennukem, of course.
Our roadmap progress is a bit slow in this moment, we have few helping hands right now, there are some improvements that we are working on at the standard level in the metadata, UI improvements and performance.

If you are interested in joining to improve the project, please join to our #watchit slack channel https://synapse-media.slack.com/ssb/redirect

**Comment by bennukem** [comment]
I am not used to use slack , do I have to give you my email to participate ?
I guess I can have a look even if I am not sure I can help a lot with javascript. Have to see, I guess

**Reply by geolffreym** [reply]
Definitely not!! we can follow up in "discussions".. let me know how can i help :)
